linux/vhba_hot_add_remove/freebsd_create_disk_partition.yml

```diff
@@ -32,13 +32,17 @@
   when: partition_fstype is not in ['ufs', 'zfs']
 
 - name: "Create partition table on disk {{ disk_name }}"
-  ansible.builtin.shell: gpart create -s gpt {{ disk_name }}
+  ansible.builtin.command: gpart create -s GPT {{ disk_name }}
   delegate_to: "{{ vm_guest_ip }}"
 
 - name: "Create 1 partition on disk {{ disk_name }}"
   ansible.builtin.shell: gpart add -t freebsd-{{ partition_fstype }} {{ disk_name }}
   delegate_to: "{{ vm_guest_ip }}"
 
+- name: "Show partition table on disk {{ disk_name }}"
+  ansible.builtin.command: gpart show {{ disk_name }}
+  delegate_to: "{{ vm_guest_ip }}"
+
```

Comment:
You can put above 3 shell commands into one shell task. Refer to the last example in https://docs.ansible.com/ansible/latest/collections/ansible/builtin/shell_module.html#examples.